crates/air-parser/src/parser/air.lalrpop

```diff
@@ -82,25 +82,26 @@ PeerPart: PeerPart<'input> = {
 }
 
 Output: CallOutputValue<'input> = {
-    <s:Alphanumeric> => CallOutputValue::Scalar(s),
-    <a:Accumulator> => CallOutputValue::Accumulator(a),
+    <a:Alphanumeric> => CallOutputValue::Variable(Variable::Scalar(a)),
+    <s:Stream> => CallOutputValue::Variable(Variable::Stream(s)),
 };
 
 Function = CallInstrValue;
 PeerId = CallInstrValue;
 ServiceId = CallInstrValue;
 
 CallInstrValue: CallInstrValue<'input> = {
-    <s:Literal> => CallInstrValue::Literal(s),
-    <s:Alphanumeric> => CallInstrValue::Variable(s),
+    <l:Literal> => CallInstrValue::Literal(l),
+    <a:Alphanumeric> => CallInstrValue::Variable(Variable::Scalar(a)),
+    <s:Stream> => CallInstrValue::Variable(Variable::Stream(s)),
     <l: @L> <v:JsonPath> <r: @R> => {
-        let (variable, path) = into_variable_and_path(v.0, v.1, v.2);
```

Comment:
is it possible to have 
```
    <l: @L> <(variable, path, other):JsonPath> <r: @R> => {
```
to avoid `v.0`

Reply:
It's impossible to use comma here :(